### ISSUE TITLE
How to inference ?

### ISSUE THREAD

**Issue by zhangwanjun4843** [ChenyangQiQi/BSVD#7]
You've done excellent research, I'm having trouble inference right now, can you provide an denoising example program for a noise reduction video stream？thanks

**Comment by ChenyangQiQi** [comment]
Hi, thanks for your interest!
Does the command in https://github.com/ChenyangQiQi/BSVD#test not work on your machine?
You may give me the screenshot of the error

**Comment by zhangwanjun4843** [comment]
I use the blind method to train. When I test according to the format similar to bsvd_c64.yml, an error occurs when loading the ckpt_state of the weight
`/BSVD/Experimental_root/archs/bsvd_arch.py", line 226, in load
    self.load_state_dict(state_dict)`


`Error(s) in loading state_dict for InputCvBlock:
	size mismatch for convblock.0.weight: copying a param with shape torch.Size([30, 3, 3, 3]) from checkpoint, the shape in current model is torch.Size([16, 4, 3, 3]).
	size mismatch for convblock.0.bias: copying a param with shape torch.Size([30]) from checkpoint, the shape in current model is torch.Size([16]).
	size mismatch for convblock.3.weight: copying a param with shape torch.Size([16, 30, 3, 3]) from checkpoint, the shape in current model is torch.Size([16, 16, 3, 3]).
`

**Comment by ChenyangQiQi** [comment]
I guess you use the wrong config to test our BSVD.
30 is the default config for 'interm_ch'. However, it is better to manually set the value of  'interm_ch' during both training and inference to make there the model architecture is the same.

In addition, the output channel of your current mode is 4, which means you add the noise channel during inference, which is inconsistent with your training setting.

Basically, it is better to check the model printed in the log file and check the argument during inference to make during your model architecture is consistent
